CefSharp/IJavascriptObjectRepository.cs

```diff
@@ -13,6 +13,7 @@ namespace CefSharp
     /// </summary>
     public interface IJavascriptObjectRepository : IDisposable
     {
+        string WindowPropertyName { get; set; }
```

Comment:
There will be multiple settings to add, so an object that holds the settings and performs the validation is what I can in mind.

Reply:
I have now created a settings object currently only containing the `WindowPropertyName`.